data-prepper-plugins/http-source/src/test/java/org/opensearch/dataprepper/plugins/source/loghttp/HTTPSourceConfigTest.java

```diff
@@ -278,17 +75,13 @@ void isPathValid_should_return_false_for_invalid_path() throws NoSuchFieldExcept
         assertThat(objectUnderTest.isPathValid(), equalTo(false));
     }
 
-        private void reflectivelySetField(final HTTPSourceConfig httpSourceConfig, final String fieldName, final Object value) throws NoSuchFieldException, IllegalAccessException {
-            final Field field = HTTPSourceConfig.class.getDeclaredField(fieldName);
-            try {
-                field.setAccessible(true);
-                field.set(httpSourceConfig, value);
-            } finally {
-                field.setAccessible(false);
-            }
-        }
-
-        private String getS3FilePath() {
-            return S3_PREFIX.concat(UUID.randomUUID().toString());
+    private void reflectivelySetField(final HTTPSourceConfig httpSourceConfig, final String fieldName, final Object value) throws NoSuchFieldException, IllegalAccessException {
```

Comment:
This was added as a test helper. Let's use that:

https://github.com/opensearch-project/data-prepper/blob/6648d9fea792cfca889c69f62ce50af6ee12392a/data-prepper-test-common/src/main/java/org/opensearch/dataprepper/test/helper/ReflectivelySetField.java#L29-L38